### DISCUSSION TITLE
The Hybrid (STAGE 2: Testing)

### DISCUSSION THREAD

**Discussion by nasko222** [nasko222/the-minecraft-recreation-project#167] [Show and tell]
# UPDATE: I am no longer updating Hybrid in this discussion and we moved in #the-hybrid channel of the new TMRP Discord Server - https://discord.gg/Js4FnZ2fcf Join if you want to access Hybrid builds above v0.10.0.0 and the new Calendar Mode

The Hybrid project is a minecraft mod where there's a toggleable option for every single change between Beta 1.7 and Beta 1.8. This project started as a "Have one project for every Beta 1.8 dev bulld". And still that's the main idea of it. To compile every Beta 1.8 dev build with accuracy and quality. Ever since BlueStaggo made that b1.8 Issue I have been interested to make this, and now the project is seperated into 3 stages

Stage 1. Development, And because I am very close to finishing every single toggle, (Theres only one left and its of course strongholds), I decided to write this thread. A downloadable copy of The Hybrid will be posted after the development stage, I predict March this year.

Stage 2. Testing, lots and lots of testing. First I want to make sure it is crash prone to random unexpected crashes. If a crash doesn't happen in b1.7, or b1.8 that means it was The Hybrid and it has to be fixed. I prioritize crash bugs at first. Second are bugs that potentially generate incorrect worlds. Heres an example. Earlier in The Hybrid development I tried seed 404 and it generated completely different caves not the ones we all know from the 404 challenge. It so happened that I had to change the cave code a bit for b1.7 terrain. Same happened with Clay generation. Also another example the trees in the Title panorama seed were in the wrong locations  That happened because I had Mineshafts on, which scrambled the random generator and while that isn't a bug because of course Mineshafts didn't exist in b1.7 i thought its worth mentioning. Third, generation bugs with the dev strongholds and villages. Heres the thing, Structures are NOT terrain aware, this is how notch coded it, and I can add 1 million exceptions theres gonna be one seed that is gonna generate stronghold or village on water, it is what it is, But i made sure those are boiled down to the minimal, unlike previous b1.8 devs I have made. And last but not least rendering bugs. Dont get me wrong  Adding two renderers making them toggleable and work synchronously isnt an easy task. I had some light rendering issues of two renderers mixing together creating weird and foggy effect or lighting completely stopping, But I fixed most if not all of those

Stage 3: Compiling all beta 1.8 devs, After all testing, its time to compile all b1.8 developer and trailer builds, Hybrid will be out of debug mode and the toggles will be pre-determined for each build. This is also the stage where I plan to release the Source code of The Hybrid and allow modders to use it, and credit me and Staggo for it.

The development of The Hybrid is soon gonna come to an end, So signup down here if you want to be a tester when its out!

**Comment by TechGuysTechSpotOfficial** [comment]
I would like to join testing, the main reason being is well I want to learn from my mistakes and show I can be a good person, I promise to not distribute anything and I will report any bugs if I see any. This is a really awesome project and I am really interested in beta 1.8 development, I haven't tested any of your stuff in ages and I want to relive that experience again even if it's just one last time

**Comment by MoonlightSystem** [comment]
I would love to test this! I always have fun breaking mods/games to help improve the final product.

**Comment by AnnoyedUser420** [comment]
I would like to join testing

**Comment by Nicky2026-21** [comment]
I guess I can join testing The Hybrid just in case things wouldn't get worse from here.

**Reply by nasko222** [reply]
Don't even think about it

**Comment by otrexdev** [comment]
Great Job Nasko222!!

**Reply by nasko222** [reply]
FUCK YOU :DDDDDDDDDDDDDDDDDDDD

**Comment by TNAZEP** [comment]
Would love to test this out, always been very interested in the between 1.7 - 1.8 builds!

**Comment by forkiesassds** [comment]
beta 1.8.x at home :trollface:

**Reply by nasko222** [reply]
beta 1.8.x is mod, this one is one big fat remake, also both of those got some staggo code :trollface:

**Comment by nasko222** [comment]
Yo. I didnt expect that much interest, Well here are 150 images, my screenshots folder as of like start of 2024 (when i moved the project to my new pc) There are probably more screenshots but im too lazy to find them lol. Most of these are villages or farlands or bugs, but idk you might find it interesting

https://naskogdps17.7m.pl/recreationstation/hybriddevscreenshots.zip

**Reply by TechGuysTechSpotOfficial** [reply]
doesn't work, it leads to an image called nohotlink.png

**Reply by nasko222** [reply]
I blame it on the hosting

**Reply by nasko222** [reply]
Alternate link until 7m fixes their shit - https://www.mediafire.com/file/hn1fm82m8dnt8l8/hybriddevscreenshots.zip/file

good ol mediafire

**Reply by TechGuysTechSpotOfficial** [reply]
thanks, now it works

**Comment by GameMasterPDX** [comment]
i would like to test….

**Comment by nasko222** [comment]
Well thank you for all being interested in testing this. I didn't expect that many to be interested in yet another big fat "remake" (as some might say). Uhh.. Unfortunately there's slight change of plans, I'm going on two week vacation because I felt like it, and I might not make it for March. Well in reality the only thing thats currently left to do are Surface strongholds. But I can't do it alone, Many of you know I tried over the years, and I did try again with a new system and things aren't looking pretty. That said, I got someone who might help, but I need to wait a few more weeks for that too, so currently I am debating myself whether to release The Hybrid without the Surface stronghold addition, and let you all testers do your testing thing. Thoughts?

**Reply by TechGuysTechSpotOfficial** [reply]
I don't mind if you release it without the surface strongholds, we can test out the other features until then, like I said in my post I will report any bugs if I see any

**Reply by GameMasterPDX** [reply]
i agree with techguy

**Comment by nasko222** [comment]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/52720373/3a60a63b-9513-4397-8271-17a88dd08ffc)
Also I'm bad with archiving this but old builds are buggy xd

**Reply by G23M08** [reply]
Release the old hybrid versions when the full one gets relesed

**Reply by nasko222** [reply]
I will release the source once it gets released and all versions will be able to be compiled and downloaded, fun!

**Reply by nasko222** [reply]
old ones are insanely crashy though

**Comment by nasko222** [comment]
Updates, I am back from vacation but i'm kinda sick. Strongholds progress somewhat

Well I'll release it next week, strongholds or not, there will be post 1.0 updates here anyway, stay tuned

**Reply by GameMasterPDX** [reply]
its been two weeks

**Reply by nasko222** [reply]
Please be patient as Technical difficulties arised

**Comment by nasko222** [comment]
Testing stage has begun, check the top post for more information.

**Comment by 483aa** [comment]
I'll test it tomorrow if I can.

**Comment by G23M08** [comment]
what does the herobrine option do

**Comment by 483aa** [comment]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/159472635/7ad15259-56fd-4812-8133-b0f1c88ede54)
what its

**Reply by 483aa** [reply]
so cool named

**Comment by G23M08** [comment]
![Captura de tela 2024-03-18 134119](https://github.com/nasko222/the-minecraft-recreation-project/assets/147005383/bbcf3d16-a07c-4e5e-abb6-b071e52fad33)
 what

**Reply by 483aa** [reply]
lol

**Reply by TechGuysTechSpotOfficial** [reply]
did you not set your username to your mc name lol

**Comment by TechGuysTechSpotOfficial** [comment]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/105efc06-d631-4f15-92dd-6bfd038f5154)
Pretty cool 2 dev village seed and here's my terrain config to get it
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/ca83faae-420f-4b94-95bd-b9d5cb2cfe9d)

**Comment by 483aa** [comment]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/159472635/3c4b3f07-2a1f-4321-9a99-c659badad340)
farlands!!!!!!!!!

**Reply by 483aa** [reply]
so lagged

**Reply by TechGuysTechSpotOfficial** [reply]
yep you can enable the farlands

**Reply by TechGuysTechSpotOfficial** [reply]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/462d04ab-24fc-4d45-bc5e-3d31c9b8b27a)
it looks pretty sick ngl

**Reply by G23M08** [reply]
what happends if uy make your screen smaller  like it hshows some blue text

**Reply by TechGuysTechSpotOfficial** [reply]
wdym

**Reply by G23M08** [reply]
it shows what setting you are using

**Reply by TechGuysTechSpotOfficial** [reply]
oh
you can enable that by holding the left arrow and moving it with the other arrow keys

**Reply by TechGuysTechSpotOfficial** [reply]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/d689cee5-a58f-4faa-9f05-866ea13a2cf1)
I have it enabled atm but I can easily disable it by moving it to the right and hiding it

**Comment by TechGuysTechSpotOfficial** [comment]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/0fb04267-f9cd-43e2-8daf-18793f35d03f)
pack.png on the hybrid

**Comment by GameMasterPDX** [comment]
is it possible to run this in multimc/prism launcher? im on steam deck so i dont think the batch file would work for me

**Reply by TechGuysTechSpotOfficial** [reply]
I don't think you can run this on mmc/prism launcher

**Reply by GameMasterPDX** [reply]
why wouldnt you be able to, its just a minecraft mod

**Reply by TechGuysTechSpotOfficial** [reply]
idk I would ask nasko but afaik the only way to run it is via the bat file

**Reply by nasko222** [reply]
currently its not just a minecraft mod, its attached to a debugger

**Reply by GameMasterPDX** [reply]
well if i cant run it in mmc/prism i probably cant test it :(

**Reply by nasko222** [reply]
bat file just calls java with parameters, if you open the command inside the bat and run it through terminal it should work

**Reply by TechGuysTechSpotOfficial** [reply]
do you have a windows pc or is your only pc related device a steam deck

**Reply by GameMasterPDX** [reply]
steam deck

**Reply by TechGuysTechSpotOfficial** [reply]
or well what nasko said could work too

**Reply by TechGuysTechSpotOfficial** [reply]
try running it via terminal

**Comment by TechGuysTechSpotOfficial** [comment]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/ee186385-36ec-4615-b8a6-f5719d1fefd1)
panorama has never looked better

**Comment by G23M08** [comment]
can anyone post the presets for specific minecraft dev versions?

**Reply by TechGuysTechSpotOfficial** [reply]
github doesn't support dat files so I can't post my presets

**Reply by G23M08** [reply]
use another uploading site

**Reply by TechGuysTechSpotOfficial** [reply]
I already posted it here in a zip

**Comment by nasko222** [comment]
the official presets are not ready, tech, use .zip xd

**Reply by TechGuysTechSpotOfficial** [reply]
ik but I have up to jeb village and well it's probably not that accurate but it'll work for now until official presets are out

**Comment by TechGuysTechSpotOfficial** [comment]
here's my unofficial and probably kinda inaccurate dev presets
[unofficialdevpresets.zip](https://github.com/nasko222/the-minecraft-recreation-project/files/14641724/unofficialdevpresets.zip)

**Reply by G23M08** [reply]
When i try to use it. It crashes

**Reply by nasko222** [reply]
Send log and settings preset you used

**Reply by G23M08** [reply]
Dosent send log used the prsets on these files

**Reply by nasko222** [reply]
what files?

when does it crash? startup, generating a world, clicking singleplayer?

how does it crash? the minecraft crash screen? white/black screen? something else?

does it say anything in the console window

can you provide screenshots or video

**Reply by G23M08** [reply]
The files that this person uploaded

 It crashes when I start it with the launch button

White screen

No logs

I will send it later bc I'm not on pc

**Reply by nasko222** [reply]
1. Which file though?
2. Consider that these are unofficial files but still they are proven working to their computer so i wanna know if its reproducable
3. The console window should say anythihg.


Make video if possible

**Reply by G23M08** [reply]
All of them it worked normaly but when I imported these files the whole program didn't work

I will make a video in like 5 hours

It shows normal stuff I think

**Reply by G23M08** [reply]
randomly started working again

**Reply by nasko222** [reply]
can you try to reproduce the bug?

**Reply by TechGuysTechSpotOfficial** [reply]
you copy the files to your hybrid folder and it works

**Reply by TechGuysTechSpotOfficial** [reply]
thehybrid.dat file is where I have the presets named so it isn't confusing

**Reply by TechGuysTechSpotOfficial** [reply]
I couldn't talk for a while because I was doing something but now I can finally respond

**Reply by G23M08** [reply]
the names got reset after it started working so maybe it was the names

**Comment by TechGuysTechSpotOfficial** [comment]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/0473592e-5c9a-4055-87d8-63438ad46ea3)
water village lol

**Comment by TechGuysTechSpotOfficial** [comment]
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/2fd50a39-ccc5-428f-bbdd-fb5672c9c61d)
2 villages seed with a village close to the original image besides no ravine
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/d0a95558-9706-4819-889c-b8ffa37b8816)

**Comment by TechGuysTechSpotOfficial** [comment]
Seed: Glacier
![image](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/970a12be-88ef-4371-92e8-ea1b9779c952)

**Comment by G23M08** [comment]
what are the setting that make the world like 1.7 but everything else like 1.8

**Reply by TechGuysTechSpotOfficial** [reply]
biome selection

**Reply by G23M08** [reply]
like not the bimoes but the map

**Reply by G23M08** [reply]
also is there anyone to optime it cuz ive git sometimes 0 fps

**Comment by G23M08** [comment]
![Captura de tela 2024-03-20 132046](https://github.com/nasko222/the-minecraft-recreation-project/assets/147005383/cad1951f-7b96-474e-87c9-fd9d57101c97)

**Reply by TechGuysTechSpotOfficial** [reply]
modern village in beta terrain?

**Reply by G23M08** [reply]
yes but  also the distance that i am is far

**Comment by ghost** [comment]
How do you run this?

**Reply by TechGuysTechSpotOfficial** [reply]
extract the zip and run the bat file

**Comment by nasko222** [comment]
for anyone who has issues running the .bat file, you havent setup your java path, google "Java environment variables path how to" or something like that

**Comment by VesuviusVenox** [comment]
nasko where remake????????

**Reply by nasko222** [reply]
FUCK YOU METHOD!

**Reply by nasko222** [reply]
is this 0.0.0.11a-dev900001212312369????????????????????

**Comment by G23M08** [comment]
i have an idea it znot in any beta version but maybe you can add arrows that spawn mods

**Comment by TechGuysTechSpotOfficial** [comment]
I haven't played in a while mainly because I am waiting for this beauty
![Unnamed_Brick_Blocks](https://github.com/nasko222/the-minecraft-recreation-project/assets/161262870/5ede243d-1104-47f2-a56a-9be2f293bf47)

**Comment by nasko222** [comment]
sorry yall i got more busy than last week + working on a new job and shit, strongholds sadly delayed probably another few weeks...

**Reply by TechGuysTechSpotOfficial** [reply]
That's fine, take your time

**Reply by TechGuysTechSpotOfficial** [reply]
it's been a month and there hasn't been any updates

**Reply by nasko222** [reply]
I know strongholds are very interesting but I can't see myself adding them anytime soon due to how much overtime I have. There are enough things to test, plus no one from my team seems interested either.

method > NASKO FUCK YOU WHERE REMAKE!!!!!!1!1!1!1!!!!1!1

**Reply by TechGuysTechSpotOfficial** [reply]
that's the thing I'm pretty sure i've tested everything, so ig I have to wait a little more

**Comment by BlueStaggo** [comment]
In case you weren't aware already, the Beta 1.2 dev build that dropped recently contains some early ravine code. Think about using this in the remakes ;)

**Reply by nasko222** [reply]
You can have The Hybrid for now, I am really busy working on two jobs. I wont be touching this anytime soon. I also got the stronghold code from Infdev+ (With permission) that I can forward

**Comment by nasko222** [comment]
![image](https://github.com/user-attachments/assets/9d6c7459-39e6-4568-abf0-dcd5c9dd1422)
Boo.

**Comment by nasko222** [comment]
Well. That took... Forever I know, Honestly if it wasn't BlueStaggo I probably wouldn't have come back to this in the next year or two. I got completely burned out of this and life took me in another direction, but lets quit rambling because I present you finally a new build of TheHybrid. That update is made entierly by BlueStaggo, like full credit goes to him, I just asked him (politely) to help me with strongholds, and he cooked a little bit of extra too.

Changelog:
Surface strongholds, I am not sure if this can get any better, its the best we're possibly gonna get, sometimes you get mega lucky, sometimes it generates two stone bricks, take it or leave it
Added Cave Cut Off (B1.7 caves) toggle
Added Ravine Type, cycle between prerelease, dev (from the newly found b1.2 dev build) and none
Changed wolfes to not spawn in PAX taigas
Strongholds with all biomes toggle will now has only 1/1000 chance to spawn, as they are extremely frequent otherwise, 1/1000 still makes them common enough (For surface strongholds check console log for coordinate output)
Tweaked reddish lighting
Some other small tweaks


Download: [hybrid_v0.9.5.zip](https://github.com/user-attachments/files/16228738/hybrid_v0.9.5.zip)

**Reply by TechGuysTechSpotOfficial** [reply]
Awesome!

**Comment by BlueStaggo** [comment]
Here are some configuration files that are mostly equivalent to the pre-Hybrid recreations that have been made by me: [presets.zip](https://github.com/user-attachments/files/16228831/presets.zip)

**Reply by TechGuysTechSpotOfficial** [reply]
cool stuff

**Comment by TechGuysTechSpotOfficial** [comment]
Here's a seed ya'll might enjoy
![image](https://github.com/user-attachments/assets/785122f0-643d-4014-9375-431780c87b8f)

**Comment by TechGuysTechSpotOfficial** [comment]
Another seed, i'm using this for my survival world
![image](https://github.com/user-attachments/assets/f8dbcbd3-ddce-43ab-aa54-ff117181be03)

**Comment by TechGuysTechSpotOfficial** [comment]
more presets by me, this combines bluestaggo's presets and mine and it also includes an extra of what my perfect beta 1.8 version would be
[unofficialdevpresets2.zip](https://github.com/user-attachments/files/16240433/unofficialdevpresets2.zip)

**Comment by nasko222** [comment]
![image](https://github.com/user-attachments/assets/7d43b9c1-8531-4513-8630-63800dcb9def)
If you activate DoNotTap (change 0 to 1) You will make the preset at the corresponding index ineditable.

**Reply by nasko222** [reply]
![image](https://github.com/user-attachments/assets/3d360670-9a5a-494a-ad40-27f6cff39032)

**Reply by TechGuysTechSpotOfficial** [reply]
I'm aware

**Comment by TechGuysTechSpotOfficial** [comment]
Another Update, this just adds white enderman to my perfect beta 1.8 version preset
[unofficialdevpresets2.1.zip](https://github.com/user-attachments/files/16322402/unofficialdevpresets2.1.zip)

**Comment by TechGuysTechSpotOfficial** [comment]
Another update, this updates my white enderman preset by using staggo's green enderman preset as a base
[unofficialdevpresets2.2.zip](https://github.com/user-attachments/files/16373727/unofficialdevpresets2.2.zip)

**Comment by TechGuysTechSpotOfficial** [comment]
fixed, I forgot to change enderman to drop nothing 
[unofficialdevpresets2.2.1.zip](https://github.com/user-attachments/files/16373806/unofficialdevpresets2.2.1.zip)

**Comment by TechGuysTechSpotOfficial** [comment]
another update, this just adds gui improvements to my perfect beta 1.8 version preset, hopefully the last update for now
[unofficialdevpresets2.2.2.zip](https://github.com/user-attachments/files/16402965/unofficialdevpresets2.2.2.zip)

**Comment by TechGuysTechSpotOfficial** [comment]
ok hopefully last update again, this just adds the new blocks and items to my perfect beta 1.8 preset
[unofficialdevpresets2.2.3.zip](https://github.com/user-attachments/files/16403243/unofficialdevpresets2.2.3.zip)

**Comment by TechGuysTechSpotOfficial** [comment]
hopefully last update, all I did was remove vines from my perfect beta 1.8 version preset since they don't generate anyway
[unofficialdevpresets2.2.3.zip](https://github.com/user-attachments/files/16407162/unofficialdevpresets2.2.3.zip)

**Comment by nasko222** [comment]
New build here, Yall might have to update your presets as some namings and stuff are changed around

# [hybrid_v0.9.73.zip](https://github.com/user-attachments/files/16919967/hybrid_v0.9.73.zip)

New:
Static mode, (generates source code that can be compiled, in order to test a "Out of hybrid" mode) which will eventually be used in the final builds
New food options, It no longer uses FoodSet but instead now its 2 toggles, removeNewFood and removeStems, removing the stems removes also the melon and removing new food restores the old drops for zombies, cows, chickens, etc
Added disableCloudDaySync which disables clouds syncing with the daytime, as it is in b1.7
Old pane texture, oldPaneTex

Changes:
Old swinging animation added to the oldInputLogic toggle
Made the explosion bigger for exploding arrow mode
Added Surface stronghold generation to the debugger, it will display coordinates like in villages
Villages should now avoid generating in lines, as new system is implemented that spreads the houses in different ways, It needs massive testing for weird scenarios
Village Doors should now not generate near glass
Removed enderman glowing eyes on 
whiteender
Added the old raw chicken texture to oldChickenTex

Fixes:
Fixed buncha scenarios with strongholds and ravines being weird
Fixed bug with panorama experience bar drawing incorrectly sometimes

**Comment by nasko222** [comment]
> (Now lets post this in the correct place)

Also I started making this table, I made it in 2 AM so there might be mistakes

https://docs.google.com/spreadsheets/d/12kj73JP9qFCjSkrMB_lPr3UIoEnPsTw83Gd_STU4iMQ

**Comment by BlueStaggo** [comment]
Here's all of the presets so far in zip form: [presets.zip](https://github.com/user-attachments/files/16922170/presets.zip)

**Comment by nasko222** [comment]
Hey there, The new hybrid ver isn't very interesting to update it here, just few texture swaps, I scrapped the B166 behavioral too

But there's something for testing, I have went ahead and compiled b1.7.3 and b1.8-pre1 out of the Hybrid, These two versions should have ZERO side effects of their respective, the b1.7.3 should act like b1.7.3 and the b1.8-pre1 should act like b1.8-pre1, No crashes (except when it is expected due to bug in the real version), No b1.8 stuff in b1.7 and no b1.7 stuff in b1.8 and so on.

Here you go testers

[hybrid_compile_test_0975.zip](https://github.com/user-attachments/files/16966930/hybrid_compile_test_0975.zip)

**Comment by TechGuysTechSpotOfficial** [comment]
I slightly updated the official dev presets, to be more closer to what I think it is, I mainly modified the first few, and btw here's an image of the changes I did 
![image](https://github.com/user-attachments/assets/e933976a-d536-479b-974e-2f9706e48862)
[SlightlyAlteredDevPresetsV1.0.zip](https://github.com/user-attachments/files/16978717/SlightlyAlteredDevPresetsV1.0.zip)

**Reply by TechGuysTechSpotOfficial** [reply]
Update: Strongholds are now added in earlier versions (Screenshot Saturday and Oh Shoot)
[SlightlyAlteredDevPresetsV1.1.zip](https://github.com/user-attachments/files/16982910/SlightlyAlteredDevPresetsV1.1.zip)

**Reply by TechGuysTechSpotOfficial** [reply]
Mineshafts added in oh shoot and notch structures based on the cave spiders date, ik this is getting kinda closer to the official dev presets but without the new food
[SlightlyAlteredDevPresetsV1.2.zip](https://github.com/user-attachments/files/16993036/SlightlyAlteredDevPresetsV1.2.zip)

**Comment by nasko222** [comment]
# [hybrid_v0.9.83.zip](https://github.com/user-attachments/files/17000099/hybrid_v0.9.83.zip)

Moved everything that has the opposite polarity to "Experimental" tab, how b1.8 should look is everything is false and every bool setting is the first in the menu list, b1.7 - everything true and last bool setting in menu list (EXCEPT THE EXPERIMENTAL, which should be again false and first setting)

Quite some stuff got renamed, so your presets may need to be redone.

Debugger changed a little, now you can move stuff with left and right arrow key and they appear in columns

Fixed chicken texture polarity
Unlicensed copy is now a boolean
Added old lighting color toggle
Removed client interpolation and hybrid will now use B173 interpolation always
Blank Fence gate and Blank vines are now combined in FenceGateStatus and VineStatus
Strongholds and Villages biomes are now bools
Removed some unused stuff, middle mouse button only works with b1.7 input logic and creative reach is always on, and something else ive forgot

Removed herobrine

**Comment by TechGuysTechSpotOfficial** [comment]
dev presets based on the spreadsheet as of the time I am writing this which is 9/14/2024
[latestdevpresets.zip](https://github.com/user-attachments/files/17002899/latestdevpresets.zip)

**Reply by TechGuysTechSpotOfficial** [reply]
PS: Let me know if I got certain things inaccurate

**Comment by kylethatkees** [comment]
What's happening here?
![2024-09-18_09 27 57](https://github.com/user-attachments/assets/8716462e-0540-45e9-9484-017b7450184f)

**Reply by BlueStaggo** [reply]
It's just debug text. Press the arrow keys to cycle it through.

**Comment by Survivor71Z** [comment]
I'd love to be a tester for The Hybrid, experimenting with old versions of Minecraft is what I do 90% of the time!

**Reply by TechGuysTechSpotOfficial** [reply]
there's a download in the top post of the discussion

**Reply by Survivor71Z** [reply]
Ohhhh, I thought there was a verification system.

**Comment by nasko222** [comment]
Any new Hybrid topics will now be moved to the new TMRP discord server

Invite link - https://discord.gg/jQxU7c3k4X
